experiment/measurer/coverage_utils.py

```diff
@@ -271,26 +271,29 @@ def generate_json_summary(coverage_binary,
     return result
 
 
-def extract_covered_regions_from_summary_json(summary_json_file):
-    """Returns the covered regions given a coverage summary json file."""
-    covered_regions = []
+def extract_covered_branches_from_summary_json(summary_json_file):
+    """Returns the covered branches given a coverage summary json file."""
+    covered_branches = []
     try:
         coverage_info = get_coverage_infomation(summary_json_file)
         functions_data = coverage_info['data'][0]['functions']
-        # The fourth number in the region-list indicates if the region
-        # is hit.
-        hit_index = 4
-        # The last number in the region-list indicates what type of the
-        # region it is; 'code_region' is used to obtain various code
-        # coverage statistic and is represented by number 0.
+
+        # The fourth and the fifth item tell whether the branch is evaluated to
+        # true or false respectively.
+        hit_true_index = 4
+        hit_false_index = 5
+        # The last number in the branch-list indicates what type of the
+        # region it is; 'branch_region' is represented by number 4.
         type_index = -1
-        # The number of index 5 represents the file number.
-        file_index = 5
+        # The number of index 6 represents the file number.
+        file_index = 6
         for function_data in functions_data:
-            for region in function_data['regions']:
-                if region[hit_index] != 0 and region[type_index] == 0:
-                    covered_regions.append(region[:hit_index] +
-                                           region[file_index:])
+            for branch in function_data['branches']:
+                if branch[hit_true_index] != 0 or branch[
+                        hit_false_index] != 0 and branch[type_index] == 4:
```

Comment:
according to [llvm region kind](https://llvm.org/docs/CoverageMappingFormat.html#pseudo-counter): 4 represents branch metric.

Reply:
Can you make a variable to store this magic number?